recipes/mold/all/conanfile.py

```diff
@@ -0,0 +1,96 @@
+        if self.settings.os == "Windows":
+            raise ConanInvalidConfiguration(f'{self.name} can not be built on {self.settings.os}.')
+        if self.settings.compiler == "gcc" and tools.Version(self.settings.compiler.version) < "11":
+            raise ConanInvalidConfiguration("GCC version 11 or higher required")
+        if (self.settings.compiler == "clang" or self.settings.compiler == "apple-clang") and tools.Version(self.settings.compiler.version) < "12":
+            raise ConanInvalidConfiguration("Clang version 12 or higher required")
+        if self.settings.compiler == "apple-clang" and "arm" in self.settings.arch :
+            raise ConanInvalidConfiguration('Use apple-clang does not work on ARM with this recipe')
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def _get_include_path(self, dependency):
+        include_path = self.deps_cpp_info[dependency].rootpath
+        include_path = os.path.join(include_path, "include")
+        return include_path
+
+    def _patch_sources(self):
+        if self.settings.compiler == "apple-clang":
+            tools.replace_in_file("source_subfolder/Makefile", "-std=c++20", "-std=c++2a")
+
+        tools.replace_in_file("source_subfolder/Makefile", "-Ithird-party/xxhash ", "-I{} -I{} -I{} -I{} -I{}".format(
+        self._get_include_path("zlib"),
+        self._get_include_path("openssl"),
+        self._get_include_path("xxhash"),
+        self._get_include_path("mimalloc"),
+        self._get_include_path("onetbb")
+        ))
+
+        tools.replace_in_file("source_subfolder/Makefile", "MOLD_LDFLAGS += -ltbb", "MOLD_LDFLAGS += -L{} -ltbb".format(
+            self.deps_cpp_info["onetbb"].lib_paths[0]))
+
+        tools.replace_in_file("source_subfolder/Makefile", "MOLD_LDFLAGS += -lmimalloc", "MOLD_LDFLAGS += -L{} -lmimalloc".format(
+            self.deps_cpp_info["mimalloc"].lib_paths[0]))
+
+    def requirements(self):
+        self.requires("zlib/1.2.12")
+        self.requires("openssl/1.1.1q")
+        self.requires("xxhash/0.8.1")
+        self.requires("onetbb/2021.3.0")
+        self.requires("mimalloc/2.0.6")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
+
+    def build(self):
+        self._patch_sources()
+        with tools.chdir(self._source_subfolder):
+            autotools = AutoToolsBuildEnvironment(self)
+            autotools.make(target="mold", args=['SYSTEM_TBB=1', 'SYSTEM_MIMALLOC=1'])
+
+    def package(self):
+        self.copy("LICENSE", src=self._source_subfolder, dst="licenses")
+        self.copy("mold", src=self._source_subfolder, dst="bin", keep_path=False)
+
```

Comment:
```suggestion

    def package_id(self):
        del self.info.settings.compiler

```
https://github.com/conan-io/conan-center-index/blob/master/docs/packaging_policy.md#settings

Reply:
I had this already deleted before, it fails in the CI.

Reply:
Fails how? If you are packaging only a tool, the compiler must not be part of the package ID.